dessia_common/workflow/blocks.py

```diff
@@ -1282,14 +1392,14 @@ class Archive(Block):
     :param name: Name of the block.
     """
 
-    def __init__(self, number_exports: int = 1, filename: str = "archive", name: str = "",
-                 position: Tuple[float, float] = None):
+    def __init__(self, number_exports: int = 1, filename: str = "archive", name: str = "Archive",
+                 position:  Position = (0, 0)):
         self.number_exports = number_exports
         self.filename = filename
         self.extension = "zip"
         self.text = False
         inputs = [Variable(name=f"Export {i}") for i in range(number_exports)]
-        inputs.append(TypedVariableWithDefaultValue(type_=str, default_value=filename, name="Filename"))
+        inputs.append(Variable(type_=str, default_value=filename, name="Filename"))
         Block.__init__(self, inputs=inputs, outputs=[Variable(name="Archive")], name=name, position=position)
```

Comment:
A lot of init use `NameOfClass.__init__()`. Does it have a reason? If not it's preferable to use `super().__init__`

Reply:
You're right ! For a lot of cases, it is old code legacy, but for the new ones, that is just me being careless. I tend to use `super().__init__()` more often, but sometimes I just slip into `ClassName.__init__()`.

I'll perform some changes